pnpm-workspace.yaml

```diff
@@ -185,3 +186,5 @@ onlyBuiltDependencies:
   - web3
   - web3-bzz
   - web3-shh
+ignoredBuiltDependencies:
```

Comment:
You should add this to `onlyBuiltDependencies`, not `ignoredBuiltDependencies`.  Adding it here will cause it to break in any environment where the binaries aren't already present.